reporter/symbol_uploader.go

```diff
@@ -182,7 +189,7 @@ func (d *DatadogSymbolUploader) GetExistingSymbolsOnBackend(ctx context.Context,
 }
 
 // Returns true if the upload was successful, false otherwise
-func (d *DatadogSymbolUploader) upload(ctx context.Context, uploadData uploadData) bool {
+func (d *DatadogSymbolUploader) upload(ctx context.Context, uploadData uploadData, ind int) bool {
```

Comment:
The `upload` function is incorrectly named since it's also looking for symbols on the disk. We might want to separate the part looking for symbols, to be common between different endpoints, and rename this function.